lib/console.js

```diff
@@ -65,6 +65,11 @@ function Console(logger) {
     logger.log(proc.color(this.pad(stamp,this.padding)), colors.bright_cyan(string));
```

Comment:
This `.Info()` method is no longer used, as far as I can tell. Probably a good time to delete it.